docs/enhancement-proposals/off-ep-0002.md

```diff
@@ -0,0 +1,85 @@
+# OFF-EP 0002 — Support vdW interactions with a hard cut-off
```

Comment:
The title of this OFF-EP is inconsistent with its stated purpose: It should be something like "Support cutoff-based vdW without long-range corrections".